packages/base/src/widget.ts

```diff
@@ -138,6 +137,20 @@ export class JupyterCadPanel extends SplitPanel {
     this._view.delete('axes');
   }
 
+  get wireframe(): boolean {
+    return this._view.get('wireframe') as boolean;
+  }
+
+  set wireframe(value: boolean) {
+    if (this.clipView?.enabled === true) {
+      console.warn(
+        'Cannot toggle Wireframe view while Clip Plane is enabled. Please disable the Clip Plane first.'
+      );
+      return;
+    }
```

Comment:
Sorry for suggesting this, it seems it actually works great without further action from us!

```suggestion
```

Reply:
No worries, it sounds good.